AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Core/DataStoreList+LazyLoad.swift

```diff
@@ -6,28 +6,19 @@
 //
 
 import Foundation
+import Amplify
 
 /// This extension adds lazy load logic to the `List<ModelType>`. Lazy loading means
 /// the contents of a list that represents an association between two models will only be
 /// loaded when it's needed.
-extension List {
+extension DataStoreList {
 
     /// Represents the data state of the `List`.
     internal enum LoadState {
         case pending
         case loaded
     }
 
-    // MARK: - Asynchronous API
-
-    /// Trigger `DataStore` query to initialize the collection. This function always
-    /// fetches data from the `DataStore.query`.
-    ///
-    /// - seealso: `load()`
-    public func load(_ completion: DataStoreCallback<Elements>) {
-        lazyLoad(completion)
-    }
```

Comment:
This is moved over to DataStoreList.swift, that subclasses List, to prevent API breaking change